### ISSUE TITLE
Pasting after rather than before current character

### ISSUE THREAD

**Issue by jghauser** [nvim-telescope/telescope-bibtex.nvim#23]
First of all, thanks a ton for this plugin, I've been using it for a while and on a daily basis -- and it works brilliantly!

However, there is one pet peeve I have, namely that the entries seem to be pasted before rather than after the current letter (so in functionality it's like vim's `P` rather than `p`). Can we change this or make it configurable? I almost always use `p` to paste things and the one other text inserting Telescope function I use (`Telescope symbols`) also does a `p` pasting.

I think the behaviour is because you've opted for 
```
vim.api.nvim_put({entry}, "", false, false)
vim.api.nvim_feedkeys("la", "n", true)
```
rather than

```
vim.api.nvim_put({entry}, "", true, true)
vim.api.nvim_feedkeys("a", "n", true)
```

By setting the third parameter to `true`, `p`-type pasting is employed. (I've also set the fourth parameter to `true`, which means the `l` in the second line isn't needed).

I think `p`-type pasting is what people generally expect, but if you think otherwise, that's of course fine. I think I should be able to somehow override the default behaviour although I do find telescope configuration a little daunting.

**Comment by noahares** [comment]
Thanks for pointing this out. I also think that `p`-type is the way to go. I use the plugin mostly in insert-mode while writing and there I did not notice this. I think I had the 3rd and 4th parameters set differently from what they are now and changed it for some reason (I do not remember why, but I can check). 

I plan on working on some issues this weekend and since this is very straight forward, this will be on my radar as well. In the same step I want to put the user back into the mode they called the plugin from, so this goes in a similar direction.

**Comment by noahares** [comment]
I implemented this in #27. Let me know if you notice any issues :)
